app/defines/random.h

```diff
@@ -36,6 +38,22 @@ unsigned int get_random_range(int min, int max) {
     return (get_random_int() % range) + min;
 }
 
+// set hw lfsr csr
+void set_hw_lfsr_seed(int seed) {
+    write_custom_lfsr(seed);
+}
+
+// generate 32bit pseudo random number from hardware
+unsigned int get_hw_random_int() {  
+    return read_custom_lfsr();
+}
+
+// generate pseudo random number between 0 - 127 from hardware
+unsigned int get_hw_random_0_127(){
+
+    return (read_custom_lfsr() & 127);
```

Comment:
I think we should do the same in both ```get_hw_random_0_127()``` and the ```get_random_0_127 ```
regarding achieving the 0-127 value..


Meaning:
```get_random_int() % 128;```
vs 
```read_custom_lfsr() & 127;```

And we should have similar ```get_random_range()``` for the hw version.

I would like to see if we use the same seed for both versions, we get the exact same results from the SW & the HW LFSR

Reply:
geberally speaking both lines behaves the same, as what I see from the test bench I have created it return the same numbers.